src/gluonts/mx/model/deepstate/_estimator.py

```diff
@@ -27,7 +27,7 @@
     ValidationDataLoader,
 )
 from gluonts.env import env
-from gluonts.model.deepstate.issm import ISSM, CompositeISSM
+from gluonts.mx.model.deepstate.issm import ISSM, CompositeISSM
```

Comment:
This should be a relative import

```suggestion
from .issm import ISSM, CompositeISSM
```

Then we don't have to change it whenever we move this around.